packages/video_player/README.md

```diff
@@ -40,6 +40,18 @@ Ensure the following permission is present in your Android Manifest file, locate
 
 The Flutter project template adds it, so it may already be there.
 
+The Android ExoPlayer dependency requires Java 8 desugaring to be enabled. Ensure that this
```

Comment:
We should add something about the build error that would be produced if this isn't present and how to fix it.  Something like:

If you get a build error for Android after adding this plugin like the following:

```
Note: /Users/you/.pub-cache/hosted/pub.dartlang.org/video_player-0.8.0/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayerPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
D8: Invoke-customs are only supported starting with Android O (--min-api 26)
D8: Invoke-customs are only supported starting with Android O (--min-api 26)
D8: Invoke-customs are only supported starting with Android O (--min-api 26)
D8: Invoke-customs are only supported starting with Android O (--min-api 26)
D8: Default interface methods are only supported starting with Android N (--min-api 24): void com.google.android.exoplayer2.Player$EventListener.onLoadingChanged(boolean)
com.android.builder.dexing.DexArchiveBuilderException: com.android.builder.dexing.DexArchiveBuilderException: Failed to process /Users/you/.gradle/caches/transforms-1/files-1.1/exoplayer-core-2.9.1.aar/8aee84c7c5dc1a164c4cf7db699bd6d5/jars/classes.jar
...
```

You'll need to add the following to your Gradle configuration, located in `<project root>/android/app/build.gradle`: